AncientReligions/localisation/ACR_druidic.csv

```diff
@@ -11,10 +11,10 @@ GOD_LUGH;Lugh;;;;;;;;;;;;;x
 GOD_MACHA;Macha;;;;;;;;;;;;;x
 GOD_DANA;Dannu;;;;;;;;;;;;;x
 GOD_NUADA;Nuada;;;;;;;;;;;;;x
-druidic;Celtic;;;;;;;;;;;;;x
-druidic_reformed;Druidic;;;;;;;;;;;;;x
-druidic_desc;Celtic paganism is one of the last surviving branch of the ancient druidic belief systems. A pantheon of gods and a world full of spirits, the children of Dannu rule the Skies, the Earth and the Otherworld. The main gods are Cernunnos, the Horned One, Dannu and Nuada, Angus, the god of Youth, Macha, the horse goddess, the great Daghda, Manannan mac Lir, the Sea God and Lugh of the Long Spear. Defining features in-game:\n�Y*�! Can raid neighbors for loot\n�Y*�! Low-tech non-pagans have very low Supply Limit in Pagan counties\n�Y*�! Can declare Subjugation Wars against all other pagans\n�Y*�! Rulers are limited to Gavelkind Succession\n�Y*�! Rulers cannot declare Holy Wars\n�Y*�! Inferior at conversion and vulnerable to non-pagan missionaries\n�Y*�! Rulers lose Prestige if at peace for too long\n�Y*�! Celtic Paganism is a feminist religion\n�Y*�! The Celtic Pagan faith can be Reformed;FRENCH;GERMAN;SPANISH;;;;;;;;;;x
-druidic_reformed_desc;The reformed Celtic faith has a standardized liturgy etched on the many raised stones on the British Isles, telling the story of the Tuatha De Danann. It is led by the circle of the Druids with the Arch-Druid as the religious leader of the faithful. Defining features in-game:\n�Y*�! Can raid neighbors for loot\n�Y*�! The Arch-Druid can call Great Holy Wars\n�Y*�! Druidic is a feminist religion;FRENCH;GERMAN;SPANISH;;;;;;;;;;x
+celtic_pagan;Celtic;;;;;;;;;;;;;x
+celtic_pagan_reformed;Druidic;;;;;;;;;;;;;x
+celtic_pagan_desc;Celtic paganism is one of the last surviving branch of the ancient druidic belief systems. A pantheon of gods and a world full of spirits, the children of Dannu rule the Skies, the Earth and the Otherworld. The main gods are Cernunnos, the Horned One, Dannu and Nuada, Angus, the god of Youth, Macha, the horse goddess, the great Daghda, Manannan mac Lir, the Sea God and Lugh of the Long Spear. Defining features in-game:\n�Y*�! Can raid neighbors for loot\n�Y*�! Low-tech non-pagans have very low Supply Limit in Pagan counties\n�Y*�! Can declare Subjugation Wars against all other pagans\n�Y*�! Rulers are limited to Gavelkind Succession\n�Y*�! Rulers cannot declare Holy Wars\n�Y*�! Inferior at conversion and vulnerable to non-pagan missionaries\n�Y*�! Rulers lose Prestige if at peace for too long\n�Y*�! Celtic Paganism is a feminist religion\n�Y*�! The Celtic Pagan faith can be Reformed;FRENCH;GERMAN;SPANISH;;;;;;;;;;x
```

Comment:
It needs to be _DESC for religions, otherwise the game doesn't find the localization